ffi/src/symbol_table.rs

```diff
@@ -75,6 +72,64 @@ impl Table {
     pub fn kill(&self, var: &MVAR) {
         unsafe { ST_Kill(from_ref(var).cast_mut()) };
     }
+
+    ///returns (there is data,there are descendants)
```

Comment:
This comment is the wrong way around. It should be (Var descendants are defined, Var is defined)